src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1204,6 +1211,115 @@
+    text: |
+        az acr connected-registry create --registry mycloudregistry --mode mirror
+            --parent myconnectedregistry --name mymirroracr --repository app/mycomponent
+            --sync-schedule "0 12 * * *" --sync-window PT4H --sync-audit-logs-enabled false
+            --client-tokens myTokenName1 myTokenName2
+"""
+
+helps['acr connected-registry delete'] = """
+type: command
+short-summary: Delete a connected registry from Azure Container Registry.
+examples:
+  - name: Delete a mirror connected registry 'myconnectedregistry' from parent registry 'mycloudregistry' and skip verification.
+    text: |
+        az acr connected-registry delete --registry mycloudregistry 
+            --name myconnectedregistry --yes
+"""
+
+helps['acr connected-registry install-info'] = """
+type: command
+short-summary: Retrives information required for to activate a connected registry.
+examples:
+  - name: Prints the values requiered to activate a connected registry in json format
+    text: |
+        az acr connected-registry install --registry mycloudregistry 
+            --name myconnectedregistry 
+  - name: Prints the values in json format requiered to activate a connected registry and generates the sync token credentials.
+    text: |
+        az acr connected-registry install --registry mycloudregistry 
+            --name myconnectedregistry --fresh-install
+"""
+
+helps['acr connected-registry deactivate'] = """
+type: command
+short-summary: Deactivate a connected registry from Azure Container Registry.
+examples:
+  - name: Deactivate a connected registry 'myconnectedregistry' and skip verification.
+    text: |
+        az acr connected-registry deactivate --registry mycloudregistry 
+            --name myconnectedregistry --yes
+"""
+
+helps['acr connected-registry list'] = """
+type: command
+short-summary: Lists all the connected registries under the current parent registry.
+examples:
+  - name: Lists all the connected registries of 'mycloudregistry' in table format.
+    text: >
+        az acr connected-registry list --registry mycloudregistry --output table
+  - name: Lists only the inmediate children of 'mycloudregistry' in expanded form in a table.
+    text: >
+        az acr connected-registry list --registry mycloudregistry --no-children --output table
+  - name: Lists all the offspring of 'myconnectedregistry' in expanded form inside a table.
+    text: >
+        az acr connected-registry list --registry mycloudregistry --parent myconnectedregistry --output table
+"""
+
+helps['acr connected-registry list-client-tokens'] = """
+type: command
+short-summary: Lists all the client tokens associated to a specific connected registries.
+examples:
+  - name: Lists all client tokens of 'mymirroracr'.
+    text: >
+        az acr connected-registry list-client-tokens --registry mycloudregistry --name mymirroracr
```

Comment:
Wondering, what is the use case for this? wouldn't show connected-registry provide the list?